Asterix/Main_EFC_THD.py

```diff
@@ -942,6 +937,14 @@ def correctionLoop(parameter_file,
 
     ## SAVING...
     header = useful.from_param_to_header(config)
+    if thd2.DM1.active == True:
+        cut_phaseDM1 = np.zeros((nbiter + 1, 2 * thd2.prad, 2 * thd2.prad))
+        for it in np.arange(nbiter + 1):
+            cut_phaseDM1[it] = proc.crop_or_pad_image(phaseDM1[it],
```

Comment:
why do you absolutely want to crop them to the pupil plane ? they are not that big (overpad pupil is only 25% larger than pupil). More importantly, for the DM that is outside of the pupil plane, the movement of the actuator outside the pupil is relevant because the beam is larger than the pupil. why throw away this information to save a few Mbytes ?